app/src/main/res/layout/transact_fragment_layout.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/background_card"
```

Comment:
Add `android:fillViewport="true"` to make sure that the scroll view uses the full screen even when it's not needed.

Reply:
can we leave it to other release?